### PR TITLE
Sc connext hub do not merge

### PR DESCRIPTION
**Pull request by rhlsthrm**
Tests are passing, can we merge?

### CODE REVIEW

File: modules/hub/README.rst [hunk 1]
```diff
@@ -26,6 +26,14 @@ Running Tests
 
    $ ./migrate-and-test.sh
 
+Creating Migrations
```
Comment (LayneHaber) [inline modules/hub/README.rst]: add here that you have to add it to the database module as well. we could make a precommit hook for that?

File: modules/hub/infrastructure/helm-values-production.yaml [hunk 2]
```diff
@@ -28,6 +28,18 @@ env:
   SERVICE_USER_KEY: 'nnWZrKw7WuE0K1JWb2PHup42xEVrLDnkMTcEbqecF3NeuLcleGl7crJwo2Hc0Uh'
   CHANNEL_MANAGER_ADDRESS: '0xbf2aeab03b82c182599930217272ef0fc5cf662b'
   TOKEN_CONTRACT_ADDRESS: '0x6b01c3170ae1efebee1a3159172cb3f7a5ecf9e5'
+  SHOULD_COLLATERALIZE_URL: 'https://beta.spankchain.com/api/model-signup/should-collateralize/'
```
Comment (LayneHaber) [inline modules/hub/infrastructure/helm-values-production.yaml]: do we want to commit these?

Comment (wolever) [reply modules/hub/infrastructure/helm-values-production.yaml]: Can definitely be removed

Comment (wolever) [reply modules/hub/infrastructure/helm-values-production.yaml]: (and should also be removed on our end)

Comment (rhlsthrm) [reply modules/hub/infrastructure/helm-values-production.yaml]: @wolever, is this supposed to be ignored?

File: modules/hub/README.rst (hunk 1 above)
Comment (rhlsthrm) [reply modules/hub/README.rst]: @bohendo how do you think we should handle this? We should probably have the CI process pull in migrations from the hub module into the db module.

**Comment by rhlsthrm** [comment]
Opening this PR against staging.

**Pull request closed without merging**